### DISCUSSION TITLE
Having issue with PHP installation with latest image from May 2nd (9e048184f96ae40344bbb302949ca0c725238ec5)

### DISCUSSION THREAD

**Discussion by LokeshNagireddy** [actions/virtual-environments#9796] [General questions] [answered]
Hi Team,

im getting below error while installing the latest image to solve a bug with installing AzureCLI. But im getting another issue with PHP installation.

Error info as below:
PHP version 8.3.6 (c:\tools\php\php.exe)
Found a space in the home path. We heavily discourage this, due to multiple
    incompatibilities. Please check our docs for more information on this, as
    well as some pointers on how to migrate to a different home path.
[31;1mbindgen : [31;1mThe term 'bindgen' is not recognized as a name of a cmdlet, function, script file, or executable program.[0m
[31;1m[31;1mCheck the spelling of the name, or if a path was included, verify that the path is correct and try again.[0m
[31;1m[31;1mAt C:\image\SoftwareReport\SoftwareReport.Common.psm1:53 char:5[0m
[31;1m[31;1m+     bindgen --version | Get-StringPart -Part 1[0m
[31;1m[31;1m+     ~~~~~~~[0m
[31;1m[31;1m+ CategoryInfo          : ObjectNotFound: (bindgen:String) [], ParentContainsErrorRecordException[0m
[31;1m[31;1m+ FullyQualifiedErrorId : CommandNotFoundException[0m

Can someone please help me to solve the issue.
I have already created a bug for the same.

Thank you,
Regards,
Lokesh N

**Comment by erik-bershel** [comment]
Please use the `main` branch to build images.

**Reply by LokeshNagireddy** [reply]
Hi Erik,

Im using main branch to take the commit reference. Please find the screen shot below for your reference.

![image](https://github.com/actions/runner-images/assets/113025693/79fd6182-abf5-4121-b8ef-70ab86cbb06e)

i need the fix done for Azure CLI and also dont want any build issues while installing other s/w.

Please guide me if i miss something

**Reply by erik-bershel** [reply, marked as answer]
We were blocked by the same issue. See [there](https://dev.azure.com/github/runner-images/_build/results?buildId=135601&view=logs&j=747417c8-a650-5aed-afed-d9b52dbd7d24&t=f05f5d13-c61c-51fd-2a8c-71d3e56d3e3f).
Core item: https://github.com/actions/runner-images/issues/9780
Test build [here](https://dev.azure.com/github/runner-images/_build/results?buildId=135596&view=results).
Should work fine according to your screenshot since you have `fba8994` in thread.

**Reply by LokeshNagireddy** [reply]
Thanks for your response. im rerunning my build with new commit 71da12569d6608f584c073f83ff58533c79ca4cd.
i will get back to you tomorrow..

**Reply by LokeshNagireddy** [reply]
@erik-bershel , This issue solved with latest image reference https://github.com/actions/runner-images/commit/71da12569d6608f584c073f83ff58533c79ca4cd. Thanks alot for your help.

Can you also help me in the swift install issue.

**Reply by erik-bershel** [reply]
Hey @LokeshNagireddy,

We cannot help you with the firewall settings. Only one workaround I may recommend is to use `hkps` instead of `hkp` in your branch. As you may see `hkp` is working fine in our test-repo: [Ubuntu22](https://dev.azure.com/github/runner-images/_build?definitionId=103).

**Comment by LokeshNagireddy** [comment]
Issue solved
